cnames_active.js

```diff
@@ -2069,6 +2069,7 @@ var cnames_active = {
   "null": "false.netlify.com",
   "num2words": "mcnagynorbi.github.io/num2words",
   "numberproto": "vfdan.github.io/NumberProto.js",
+  "numtoword-cli: "infysumanta.github.io/numtoword-cli/",
```

Comment:
```suggestion
  "numtoword-cli": "infysumanta.github.io/numtoword-cli",
```
You are missing a quote, and the entry doesn't need a trailing slash